integration-tests/src/test/java/org/openmrs/maven/plugins/packager/config/ValidateConfigurationsTest.java

```diff
@@ -0,0 +1,99 @@
+package org.openmrs.maven.plugins.packager.config;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.doThrow;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.Result;
+import org.junit.runner.RunWith;
+import org.openmrs.module.initializer.validator.Validator;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(Validator.class)
+public class ValidateConfigurationsTest {
```

Comment:
@mseaton I completely turned this test around. As a matter of fact it is not anymore a Mojo _integration_ test, so happy to move it in the regular test files location. Just let me know your thoughts.

Reply:
@ibacher, since this is taking advantage of the already existing test resources, we can leave it here I guess?